cmd/get/nodepools/command.go

```diff
@@ -0,0 +1,98 @@
+package nodepools
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
+)
+
+const (
+	name  = "nodepools <nodepool-id>"
+	alias = "nodepool"
+
+	shortDescription = "Display one or many node pools"
+	longDescription  = `Display one or many node pools
+
+Output columns:
+
+- ID: Unique identifier of the node pool.
+- CLUSTER ID: Unique identifier of the cluster that the node pool belongs to.
+- CREATED: Date and time of the node pool CR creation.
+- CONDITION: Latest condition reported for the node pool.
+- NODES MIN/MAX: Node pool autoscaler settings (if supported).
+- NODES DESIRED: The total number of nodes that the node pool should have.
+- NODES READY: The number of nodes in the node pool that are actually ready.
+- DESCRIPTION: User friendly description for the node pool.`
+
+	examples = `  # List all node pools you have access to
+  kgs get nodepools
+
+  # Get one specific nodepool by its ID
+  kgs get nodepools 3f01a
```

Comment:
Feels better in singular form when used with ID argument:

```suggestion
  kgs get nodepool 3f01a
```